examples/rdkafka_performance.c

```diff
@@ -1152,9 +1158,9 @@ int main (int argc, char **argv) {
 			"configuration property\n"
 			"               Properties prefixed with \"topic.\" "
 			"will be set on topic object.\n"
-			"               Use '-X list' to see the full list\n"
-			"               of supported properties.\n"
                         "  -X file=<path> Read config from file.\n"
+                        "  -X list      Show full list of supported properties.\n"
+                        "  -X <prop>    Get single property value\n"
```

Comment:
I dont think get() is supported in this tool (and doesn't need to be)

Reply:
Good one. Fixed, and also modified rdkafka_example.c to document this option as well.

(Dropped the ball on this, got too busy with other parts, sorry!)